hedwiki/score/HED_score_0.0.1.mediawiki

```diff
@@ -413,19 +413,19 @@ Reference:
+        ** Eye-opening-passive <nowiki>[Passive eye opening. Used with Findings-attributes tags and base schema Increasing/Decreasing]</nowiki>
             *** Eye-opening-passive-not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
-        ** Medication-effect-EEG <nowiki>[Medications effect on EEG. Used with Findings-attributes tags. Used with Findings-attributes tags and base schema 'Increasing'/'Decreasing']</nowiki>
+        ** Medication-effect-EEG <nowiki>[Medications effect on EEG. Used with Findings-attributes tags. Used with Findings-attributes tags and base schema Increasing/Decreasing]</nowiki>
             *** Medication-effect-EEG-not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
-        ** Medication-reduction-effect-EEG <nowiki>[Medications reduction or withdrawl effect on EEG. Used with Findings-attributes tags. Used with Findings-attributes tags and base schema 'Increasing'/'Decreasing']</nowiki>
+        ** Medication-reduction-effect-EEG <nowiki>[Medications reduction or withdrawl effect on EEG. Used with Findings-attributes tags. Used with Findings-attributes tags and base schema Increasing/Decreasing]</nowiki>
             *** Medication-reduction-effect-EEG-not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
-        ** Auditive-stimuli-effect <nowiki>[Used with Findings-attributes tags. Used with Findings-attributes tags and base schema 'Increasing'/'Decreasing']</nowiki>
+        ** Auditive-stimuli-effect <nowiki>[Used with Findings-attributes tags. Used with Findings-attributes tags and base schema Increasing/Decreasing]</nowiki>
             *** Auditive-stimuli-effect-not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
-        ** Nociceptive-stimuli-effect <nowiki>[Used with Findings-attributes tags. Used with Findings-attributes tags and base schema 'Increasing'/'Decreasing']</nowiki>
+        ** Nociceptive-stimuli-effect <nowiki>[Used with Findings-attributes tags. Used with Findings-attributes tags and base schema Increasing/Decreasing]</nowiki>
             *** Nociceptive-stimuli-effect-not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
-        ** Physical-effort-effect <nowiki>[Used with Findings-attributes tags.Used with Findings-attributes tags and base schema 'Increasing'/'Decreasing']</nowiki>
+        ** Physical-effort-effect <nowiki>[Used with Findings-attributes tags.Used with Findings-attributes tags and base schema Increasing/Decreasing]</nowiki>
             *** Physical-stimuli-effect-not-possible-to-determine <nowiki>[Not possible to determine]</nowiki>
-        ** Cognitive-tasks-effect <nowiki>[Used with Findings-attributes tags.Used with Findings-attributes tags and base schema 'Increasing'/'Decreasing']</nowiki>
+        ** Cognitive-tasks-effect <nowiki>[Used with Findings-attributes tags.Used with Findings-attributes tags and base schema Increasing/Decreasing]</nowiki>
```

Comment:
Remove the extra "Used with Findings-attribute tags." sentence at the beginning of all of the descriptions involving Increasing and Decreasing.